controllers/operands/cdi_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"time"
 
+	openshiftconfigv1 "github.com/openshift/api/config/v1"
```

Comment:
nit: group this with other openshift imports

Reply:
done